src/common.css

```diff
@@ -226,6 +226,15 @@ body.loading .page:target .password-box > .loading-spinner {
     z-index: 3;
```

Comment:
I can't mark it here, but the rule in Line 53 should state:
```
.page-header-back-button {
    width: 3rem;
    height: 3.6rem;
    position: absolute;
    left: 4rem;
    cursor: pointer;
    opacity: 0.4;
    transition: opacity .3s ease, left .3s ease;
}
```
to adjust for the `line-height: 1.2;`.